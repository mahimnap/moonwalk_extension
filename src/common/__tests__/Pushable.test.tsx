import React from "react";

import TestRenderer from "../../helpers/testRenderer";
import Pushable from "../Pushable";

describe("Pushable", () => {
  it("renders correctly", () => {
    const tree = TestRenderer(<Pushable onPress={jest.fn()} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
