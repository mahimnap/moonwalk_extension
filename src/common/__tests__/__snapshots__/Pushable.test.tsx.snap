// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pushable renders correctly 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "transform": Array [
        Object {
          "scale": 1,
        },
      ],
    }
  }
/>
`;
